docs/source/user/measurements.rst

```diff
@@ -0,0 +1,79 @@
+.. _measurements:
+
+Measurements
+===============
+
+.. contents:: |toctitle|
+    :local:
+
+Overview
+--------
+This documentation gives a high-level overview of the measurements that are available in the library.
+Refer to the :ref:`measurement` section for an explanation of what a measurement is.
+
+As covered in the :ref:`chaining` section, the inner domains and metrics need to match when chaining.
+This means you will need to choose a measurement that chains with your :ref:`aggregator <aggregators>`.
+
+In the following table, the scalar-valued and vector-valued versions of each measurement are listed separately.
```

Comment:
Maybe add a little more detail explaining that choice of `D` determines which you get.